pom.xml

```
@@ -99,6 +99,22 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-security</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-jackson</artifactId>
+            <version>0.13.0</version>
```

same for other jwt dependencies
```suggestion
            <version>${jjwt.version}</version>
```